### PR TITLE
MINOR: re-enable EosBetaUpgradeIntegrationTest

### PR DESCRIPTION
**Pull request by ableegoldman**

### CODE REVIEW

File: streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java [hunk 1]
```diff
@@ -594,7 +594,7 @@ public static void waitForStandbyCompletion(final KafkaStreams streams,
                 final List<KeyValue<K, V>> readData =
                     readKeyValues(topic, consumer, waitTime, expectedNumRecords);
                 accumData.addAll(readData);
-                assertThat(reason, accumData.size(), is(greaterThanOrEqualTo(expectedNumRecords)));
+                assertThat(reason + ",  currently accumulated data is " + accumData, accumData.size(), is(greaterThanOrEqualTo(expectedNumRecords)));
```
Comment (abbccdda) [inline streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java:597]: nit: should we also log expected records then?

**Comment by guozhangwang** [comment]
LGTM.

**Comment by vvcephei** [comment]
Hey @ableegoldman @guozhangwang , this should be merged to 2.6 as well, right?

(cc @rhauch )

**Comment by chia7712** [comment]
```EosBetaUpgradeIntegrationTest``` gets unstable recently. Maybe we should re-disable it again :(

**Comment by ableegoldman** [comment]
I believe @mjsax is planning to look into it after his current work, but it seems reasonable to disable it again until then

**Pull request merged**